dashboard/app/helpers/users_helper.rb

```diff
@@ -12,6 +12,29 @@ module UsersHelper
   ACCT_TAKEOVER_OAUTH_TOKEN = 'clever_takeover_token'
   ACCT_TAKEOVER_FORCE_TAKEOVER = 'force_clever_takeover'
 
+  # Move followed sections from old_user to new_user and destroy old_user
+  def move_sections_and_destroy_old_user(old_user, new_user)
+    # No-op if old_user is nil
+    return true unless old_user.present?
+
+    if old_user.has_activity?
+      # We don't want to destroy an account with progress.
+      # In theory this should not happen, so we log a Honeybadger error and return.
+      Honeybadger.notify(
+        error_class: 'Oauth takeover called for user with progress',
+        error_message: "Attempted to take over account with id #{old_user.id}, which has activity"
```

Comment:
Would it be useful to include the `new_user.id` in this error message?

Reply:
yeah, i like that idea